src/main/resources/io/jenkins/plugins/sample/OnboardingConfiguration/config.jelly

```diff
@@ -0,0 +1,32 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+        <f:section title="Onboarding Plugin">
```

Comment:
Unusually deep indentation with. Jelly files are usually indented 2-4 spaces, depending on the project. Be mindful of applying the same formatting you see elsewhere in a repo, in this case it would be 4 spaces per https://github.com/ajoshi-CB/jenkins-onboarding-plugin/pull/1/files#diff-7887b9f916bab35d2b74f3ab3ea03f1882dff90736039c902a7072588c46e755.